oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentStoreStatsTest.java

```diff
@@ -49,6 +74,10 @@ public void shutDown(){
         statsProvider.close();
         new ExecutorCloser(executor).close();
     }
+    @Before
+    public void startUp() {
+        stats = new DocumentStoreStats(statsProvider);
+    }
```

Comment:
Why is this needed? The instance variable `stats` already gets initialized like this.

Reply:
I will revert this change, I agree this is not required.